app/models/work_type.rb

```diff
@@ -63,63 +63,47 @@ class InvalidType < StandardError; end
-        cocina_type: Cocina::Models::ObjectType.object)
+  def to_s
+    @label
   end
 
-  def self.find(id)
-    (all + [purl_reservation_type]).find { |work| work.id == id } || raise(InvalidType, "Unknown worktype #{id}")
+  def self.find(label)
+    all.find { |work| work.label == label } || raise(InvalidType, "Unknown worktype #{label}")
   end
 
   # id is a value acceptable for MODS typeOfResource
 
   def self.all # rubocop:disable Metrics/AbcSize
     [
-      new(id: 'text', label: 'Text', html_label: 'Text', icon: 'book-open',
-          subtypes: TEXT_TYPES, cocina_type: Cocina::Models::ObjectType.object),
-      new(id: 'data', label: 'Data', html_label: 'Data', icon: 'chart-bar',
-          subtypes: DATA_TYPES, cocina_type: Cocina::Models::ObjectType.object),
-      new(id: 'software, multimedia', label: 'Software/Code', html_label: 'Software/<wbr>Code'.html_safe, icon: 'mouse',
-          subtypes: SOFTWARE_TYPES, cocina_type: Cocina::Models::ObjectType.object),
-      new(id: 'image', label: 'Image', html_label: 'Image', icon: 'images',
-          subtypes: IMAGE_TYPES, cocina_type: Cocina::Models::ObjectType.object),
-      new(id: 'sound', label: 'Sound', html_label: 'Sound', icon: 'microphone-alt',
-          subtypes: SOUND_TYPES, cocina_type: Cocina::Models::ObjectType.object),
-      new(id: 'video', label: 'Video', html_label: 'Video', icon: 'film',
-          subtypes: VIDEO_TYPES, cocina_type: Cocina::Models::ObjectType.object),
-      new(id: 'music', label: 'Music', html_label: 'Music', icon: 'music',
-          subtypes: MUSIC_TYPES, cocina_type: Cocina::Models::ObjectType.object),
-      new(id: 'mixed material', label: 'Mixed Materials', html_label: 'Mixed Materials', icon: 'play',
-          subtypes: MIXED_TYPES, cocina_type: Cocina::Models::ObjectType.object),
-      new(id: 'other', label: 'Other', html_label: 'Other', icon: 'archive',
-          subtypes: [], cocina_type: Cocina::Models::ObjectType.object)
+      new(label: 'Text', subtypes: TEXT_TYPES, cocina_type: Cocina::Models::ObjectType.object),
+      new(label: 'Data', subtypes: DATA_TYPES, cocina_type: Cocina::Models::ObjectType.object),
+      new(label: 'Software/Code', subtypes: SOFTWARE_TYPES, cocina_type: Cocina::Models::ObjectType.object),
+      new(label: 'Image', subtypes: IMAGE_TYPES, cocina_type: Cocina::Models::ObjectType.object),
+      new(label: 'Sound', subtypes: SOUND_TYPES, cocina_type: Cocina::Models::ObjectType.object),
+      new(label: 'Video', subtypes: VIDEO_TYPES, cocina_type: Cocina::Models::ObjectType.object),
+      new(label: 'Music', subtypes: MUSIC_TYPES, cocina_type: Cocina::Models::ObjectType.object),
+      new(label: 'Mixed Materials', subtypes: MIXED_TYPES, cocina_type: Cocina::Models::ObjectType.object),
+      new(label: 'Other', subtypes: [], cocina_type: Cocina::Models::ObjectType.object)
     ]
   end
 
   def self.type_list
-    all.map(&:id).sort
+    all.map(&:label).sort
```

Comment:
not blocker, but would be nice to make sure a test covers this line.

Reply:
Added.